core/app/models/spree/calculator/flat_percent_item_total.rb

```diff
@@ -9,14 +9,15 @@ def self.description
     end
 
     def compute(object)
-      computed_amount = (object.amount * preferred_flat_percent / 100).round(2)
+      compute(object)
```

Comment:
why do we want to implement this way? Let's make method calling univocal

Reply:
Are the `require_dependency 'spree/calculator'` still needed throughout the calculators? If not I can remove them on this PR if you like?